app/modules/users/actionTypes/taskSaga.js

```diff
@@ -46,6 +47,15 @@ export type FetchAction = {|
   |},
 |};
 
+export type ForkTopicAction = {|
+  ...TaskSagaAction,
+  type: typeof FORK_TOPIC,
+  payload: {|
```

Comment:
Don't forget to extend the TaskSagaAction payload too (see the other action types). (And yes, I realize that this is extremely annoying, but so far I haven't found a better way to 'extend' a flow type :/ )